src/cms/views/push_notifications/push_notification_sender.py

```diff
@@ -81,14 +81,10 @@ def get_auth_key():
         :return: FCM API auth key
         :rtype: str
         """
-        fcm_auth_config_key = "fcm_auth_key"
         auth_key = settings.FCM_KEY
-        if auth_key.exists():
-            logger.debug("Got fcm_auth_key from database")
-            return auth_key.first().value
-        logger.warning(
-            "Could not get %r from configuration database", fcm_auth_config_key
-        )
+        if auth_key and isinstance(auth_key, str):
+            return auth_key
+        logger.warning("Could not get a proper fcm_auth_key")
         return None
```

Comment:
Actually, I think if we just set the default missing key in the settings file to `FCM_KEY = None` instead of `FCM_KEY = ""`, and an existing key to a non-empty string, we could remove the `get_auth_key()` function completely. Just set the auth key to the key specified in the settings file in line 41 and everything should work as expected...
(the logging warning can also be shown in the `__init__` function if the key is `None`...

Reply:
I thought about this, but wasn't super sure, because it might be less intuitive, that the `FCM_KEY` in settings has to be a string. But I can just add this information as a comment.

Reply:
A valid auth token for the firebase service is always a string, there is no other possibility to define this value. The only ambiguity we have is how an "undefined" key should be represented.

We could also rule out any misunderstandings here if we replace all occurrences of `if self.auth_key is None:` with `if not self.auth_key:`. This way, all "falsey" values (e.g. `""`, `False`, `None`, `0`, `{}`, ...) would disable the sending of push notifications regardless their type.

Reply:
Thanks for your explanation! I implemented the changes you considered.